codebase/general/src.lib/time.1.7/src/time.c

```diff
@@ -277,3 +277,48 @@ double TimeStrToEpoch(char *text)
   /* Return epoch time in number of seconds since 00:00UT on January 1, 1970 */
   return tme;
 }
+
+
+/*-----------------------------------------------------------------------------
+;
+; NAME:
+;       dayno
+;
+; PURPOSE:
+;       Function to compute the day of the year and the number of days in the
+;       year.
+;
+; CALLING SEQUENCE:
+;       doy = dayno(year, month, day, diy);
+;
+;     Input Arguments:
+;       year          - year [1965-2014]
+;       month         - month of year [01-12]
+;       day           - day of month [01-31]
+;
+;     Output Arguments (integer pointers):
+;       diy           - number of days in the given year
+;
+;     Return Value:
+;       error code
+;
+;+-----------------------------------------------------------------------------
+*/
+
+int dayno(int year, int month, int day, int *diy)
```

Comment:
This name `dayno` is also frequently used as a variable and a structure element.  I'd recommend renaming this function.

Reply:
While I agree, is this something that will be a lot of work to check all of the places that they function name needs to be changed at?

Reply:
I can convert this to an issue.

Reply:
Done: #633